samples/nrf9160/gps/README.rst

```diff
@@ -0,0 +1,126 @@
+.. _gps_with_supl_support_sample:
+
+nRF9160: GPS sockets and SUPL client library
+############################################
+
+The GPS socket sample reads the `GPS`_ data from the `GNSS`_ socket and parses the received frames to interpret the frame contents.
+The GPS socket sample can also download the `A-GPS`_ data from a Secure User-Plane Location (SUPL) server using the SUPL client library, if the library is installed and enabled.
+See :ref:`supl_client` for information on enabling SUPL support for the sample.
+
+Overview
+********
+
+This sample operates in three different modes.
+Each of these modes depends on the state of the GPS module.
+
+Mode 1
+======
+
+This is the startup mode in which the sample waits for the GPS module to acquire a fix on the current position.
+Information on the processes is displayed on a console.
+
+Mode 2
+======
+
+The sample switches to the second mode when the GPS module has acquired at least one fix.
+In this mode the sample also displays information about the last good fix received from the GPS module.
+The displayed information includes both PVT (Position, Velocity and Time) and `NMEA`_ strings.
+
+Mode 3
+======
+
+The sample operates in this mode only if SUPL client support is enabled.
+When the sample receives an A-GPS notification from the GPS module, the sample switches from the current mode to the third mode and starts downloading the A-GPS data requested by the GPS module.
+The sample then displays the information on the screen about the download process.
```

Comment:
Again, process or progress?

Reply:
It could be both in this case actually. I think process fit better in this case.